docs/subnets/hello-world-precompile-tutorial.md

```diff
@@ -169,14 +169,18 @@ Here are some recommended resources to learn the ins and outs of the EVM:
 
 Please install the following before getting started.
 
-First install the latest version of Go. Follow the instructions [here](https://go.dev/doc/install).
+First, install the latest version of Go. Follow the instructions [here](https://go.dev/doc/install).
 You can verify by running `go version`.
 
-Set `$GOPATH` environment variable properly for Go to look for Go Workspaces. Please read
+Set the `$GOPATH` environment variable properly for Go to look for Go Workspaces. Please read
 [this](https://go.dev/doc/gopath_code) for details. You can verify by running `echo $GOPATH`.
 
+:::info
+An easy way to set GOPATH is using the command: `export GOPATH=$HOME/go`
+:::
+
 As a few things will be installed into `$GOPATH/bin`, please make sure that `$GOPATH/bin` is in your
-`$PATH`, otherwise, you may get error running the commands below.
+`$PATH`, otherwise, you may get an error running the commands below. To do that, run the command: `export PATH=$PATH:$GOROOT/bin:$GOPATH/bin`
```

Comment:
```suggestion
`$PATH`, otherwise, you may get an error running the commands below.
To do that, run the command: `export PATH=$PATH:$GOROOT/bin:$GOPATH/bin`
```